com.mudita.mudita-center.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: com.mudita.mudita-center
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: run.sh
+separate-locales: false
+rename-desktop-file: muditacenter.desktop
+rename-icon:
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=all
```

Comment:
why is network and device=all required?

Reply:
The application needs to download files from the Internet and talk to USB devices.